docs/file-formats/excel2json.md

```diff
@@ -276,3 +299,105 @@ The output of the above command, with the template files, is:
     ]
 }
 ```
+
+
+
+## The `lists` Section for `new-excel2json` and `new-excel2lists`
+
+With the [`new-excel2lists`](../cli-commands.md#new-excel2lists) command, 
+the `lists` section of a JSON project file can be created from one or several Excel files. 
+The lists can then be inserted into a JSON project file and uploaded to a DSP server.
+
+The following example shows how to create the "lists" section which are located in a directory called `listfolder`.
+The Excel files must contain the word "list" in the name. All the files containing that word will be included.
+Files that do not contain the word will not be considered.
+
+
+```bash
+dsp-tools new-excel2lists listfolder lists.json
+```
+
+The Excel sheets must have the following structure:  
+![img-excel2json-new-lists.png](../assets/images/img-excel2json-new-lists.png)
+
+
+### Formal requirements of the Excel format
+
+
+- One Excel file may contain several lists.
+- Per Excel sheet only one list is allowed. 
+  The name of the sheet is not relevant, but must be unique within one Excel file.
+- The list name must be unique in one project.
+
+
+#### Columns
+
+- "ID (Optional)"
+    - Column where a custom ID for the node or list can be entered. 
+      This ID will be the `name` in the JSON file, this is possible for list names and node names.
+      If no ID is entered the `name` will be the list node name. The ID must be unique in the entire project.
+    - If this field is left empty an ID will be created from the node name. 
+      In case of duplicate list node names, the ID will include the names of the ancestors of the node. 
+      For example:
+          - `list1:node1:node1.1` for the node `node1.1` in `list1`
+          - `list2:node1:node1.1` for the node `node1.1` in `list2`
+- "language_tag_list": These fields are for the name of the list
+  At least one language 
+- "language_tag_number": These fields are for the names of the list nodes.
+
+
+#### Languages
+
+
+- The allowed language tags are: `de`, `en`, `fr`, `it`, `rm`
+- At least one language must be used.
+- There is no limit to the number of node columns.
+- Within one list, i.e. Excel sheet, all nodes must be consistently translated into all languages.
+  
+  Correct:
+
+![img-excel2json-list-columns-correct.png](../assets/images/img-excel2json-list-columns-correct.png)
+
+  Incorrect because `de_1` is missing:
+
+![img-excel2json-list-columns-incorrect.png](../assets/images/img-excel2json-list-columns-incorrect.png)
+
+
+  
+- Within one list, i.e. Excel sheet, all nodes must be translated into the same languages.
+
+  Incorrect, `Node 1` is missing the English translation.
+
+![img-excel2json-list-translation-incorrect.png](../assets/images/img-excel2json-list-translation-incorrect.png)
+  
```

Comment:
But this one is really not correct. Such a file doesn't exist. Could you fix it?